package.json

```diff
@@ -44,7 +43,7 @@
   },
   "devDependencies": {
     "@adobe/eslint-config-helix": "1.1.4",
-    "@adobe/openwhisk-action-builder": "3.0.14",
+    "@adobe/helix-deploy": "^3.14.1",
```

Comment:
use pinned dependencies. next time use `npm add -DE @adobe/helix-deploy` to add the exact version.